packages/forms-web-app/__tests__/unit/routes/full-appeal/submit-appeal/letter-confirming-application.test.js

```diff
@@ -0,0 +1,48 @@
+const { documentTypes } = require('@pins/common');
+const { get, post } = require('../../router-mock');
+const {
+  getLetterConfirmingApplication,
+  postLetterConfirmingApplication,
+} = require('../../../../../src/controllers/full-appeal/submit-appeal/letter-confirming-application');
+const fetchExistingAppealMiddleware = require('../../../../../src/middleware/fetch-existing-appeal');
+const {
+  validationErrorHandler,
+} = require('../../../../../src/validators/validation-error-handler');
+const {
+  rules: fileUploadValidationRules,
+} = require('../../../../../src/validators/common/file-upload');
+const setSectionAndTaskNames = require('../../../../../src/middleware/set-section-and-task-names');
+
+jest.mock('../../../../../src/middleware/fetch-existing-appeal');
+jest.mock('../../../../../src/validators/common/file-upload');
+jest.mock('../../../../../src/middleware/set-section-and-task-names');
+
+describe('routes/full-appeal/submit-appeal/application-form', () => {
```

Comment:
should this be `letter-confirming-application` instead of `application-form` at the end of the route?

Reply:
resolved.